hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/commit/integration/ITestS3ACommitterMRJob.java

```diff
@@ -586,7 +586,7 @@ private MagicCommitterTestBinding() {
     protected void validateResult(final Path destPath,
         final SuccessData successData)
         throws Exception {
-      Path magicDir = new Path(destPath, MAGIC);
+      Path magicDir = new Path(destPath, MAGIC_PATH_PREFIX);
```

Comment:
is a job id getting down? it should be, for completeness

Reply:
ack. make sense!